Functions.Templates/Templates/SqlInputBinding-CSharp/readme.md

```diff
@@ -0,0 +1,7 @@
+# SQL Input Binding - C<span>#</span>
```

Comment:
Do we need the span around the #?

Reply:
All the other trigger readme.md files has the span. Ex: https://github.com/lucyzhang929/azure-functions-templates/blob/4e3f2f3a036b5bde0138142c9b02f8b0c7808b0e/Functions.Templates/Templates/BlobTrigger-CSharp/readme.md?plain=1#L1

Reply:
Yeah, I'm just wondering why and whether it's actually necessary or just a "everyone else was doing it so I might as well".

Not a huge deal regardless, but it looks pretty weird (and shouldn't do anything by itself...) so if we can remove it that'd be nice.